### ISSUE TITLE
What should the user do when the targets are running

### ISSUE THREAD

**Issue by liutiming** [wlandau/targets#244]
## Prework

* [x] Read and agree to the [code of conduct](https://github.com/wlandau/targets/blob/main/CODE_OF_CONDUCT.md) and [contributing guidelines](https://github.com/wlandau/targets/blob/main/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/wlandau/targets/issues), whether open or closed, comment on the existing thread instead of posting a new issue.

## question
Before targets, I may submit the jobs and then work on other branches of the analysis workflow.

Now that targets are running and some jobs take very long, what should I do since all branches now are included in the targets. If I open another R session, change some code and run the targets there while one target workflow is already running, be it in the interactive session or submitted with run.bash, how will it affect the workflow? What would you recommend the user to do? Thanks!

**Comment by wlandau** [comment]
`targets` is designed to have only once instance of the pipeline run at a time. If two instances of `tar_make()` run concurrently in different R sessions on the same project, this is likely to cause major problems. If you make changes to the code while the pipeline is running, the question is whether to (1) wait until the current run finishes or (2) interrupt the current run and relaunch. That depends on how many targets your latest changes invalidate. If it's a small change, (1) is probably fine. But if the changes invalidate most of your pipeline, it would probably be more expedient to go with (2). `tar_progress()` can tell you which targets are currently running and which have finished, `tar_outdated()` can tell you which targets are not up to date, and `tar_visnetwork()` can tell you a little of both.
